Source/ACE.Server/Command/Handlers/DeveloperDatabaseCommands.cs

```diff
@@ -139,5 +145,119 @@ public static List<string> OutputShortcutSQLCommand(string playerName, uint char
+                SendMessage(session, "To set your content folder, /modifystring content_folder <path>");
+                return;
+            }
+
+            var json_folder = $"{di.FullName}{sep}json{sep}weenies{sep}";
+
+            var wcid = parameters[0];
+            var prefix = wcid + " - ";
+
+            di = new DirectoryInfo(json_folder);
+
+            var files = di.Exists ? di.GetFiles($"{prefix}*.json") : null;
+
+            if (files == null || files.Length == 0)
+            {
+                SendMessage(session, $"Couldn't find {json_folder}{prefix}*.json");
+                return;
+            }
+
+            var fi = files[0];
+
+            // convert json -> sql
+            var sqlFile = json2sql(session, json_folder, fi.Name);
+            if (sqlFile == null)
+                return;
+
+            // import sql to db
+            ImportSQL(sqlFile);
+            SendMessage(session, $"Imported {sqlFile}");
+
+            // clear this weenie out of the cache
+            if (uint.TryParse(wcid, out var weenieClassId))
+            {
+                DatabaseManager.World.ClearCachedWeenie(weenieClassId);
+                //var wo = WorldObjectFactory.CreateNewWorldObject(weenieClassId);
+            }
+        }
+
+        public static string json2sql(Session session, string folder, string json_filename)
+        {
+            var json_file = folder + json_filename;
+
+            var success = LifestonedLoader.TryLoadWeenie(json_file, out var weenie);
+
+            if (!success)
+            {
+                SendMessage(session, $"Failed to load {json_file}");
+                return null;
+            }
+
+            // output to sql
+            success = LifestonedConverter.TryConvert(weenie, out var output);
+
+            if (!success)
+            {
+                SendMessage(session, $"Failed to convert {json_file}");
+                return null;
+            }
+
+            var sqlFolder = folder.Replace("json", "sql");
+
+            var di = new DirectoryInfo(sqlFolder);
+
+            if (!di.Exists)
+                di.Create();
+
+            var sqlFilename = json_filename.Replace(".json", ".sql");
+
+            var sqlFile = new StreamWriter(sqlFilename);
+            var converter = new WeenieSQLWriter();
+            converter.CreateSQLDELETEStatement(output, sqlFile);
+            sqlFile.WriteLine();
+            converter.CreateSQLINSERTStatement(output, sqlFile);
+            sqlFile.Close();
+
+            SendMessage(session, $"Converted {json_filename} to {sqlFilename}");
+
+            return sqlFilename;
+        }
+
+        public static void ImportSQL(string sqlFile)
+        {
+            var sqlCommands = File.ReadAllText(sqlFile);
+
+            using (var ctx = new WorldDbContext())
+                ctx.Database.ExecuteSqlCommand(sqlCommands);
+        }
+
+        public static void SendMessage(Session session, string msg)
```

Comment:
We already have a helper function for this. CommandHandlerHelper.WriteOutputX

Reply:
ah i wasn't aware of the existing function, updated to WriteOutput